c11threads.h

```diff
@@ -158,10 +162,27 @@ static inline int mtx_trylock(mtx_t *mtx)
 static inline int mtx_timedlock(mtx_t *mtx, const struct timespec *ts)
 {
 	int res;
+#ifdef __APPLE__
+	struct timeval now;
+	struct timespec sleeptime;
+
+	sleeptime.tv_sec = 0;
+	sleeptime.tv_nsec = 5000000; // 5 ms
```

Comment:
I don't like this arbitrary step value, but I can't offer any better alternative at this point. At least we should make this a function of ts I think, but I'm not too bothered about it.